cxx-client/cmake/BookkeepingApiConfig.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro) 
+get_filename_component(BookkeepingApi_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+find_dependency(Boost 1.56 REQUIRED COMPONENTS unit_test_framework program_options filesystem)
+find_dependency(Threads REQUIRED)
```

Comment:
@martinboulais Actually there's not much to change in here, for the moment just use same list of dependencies are in CMake files (`grpc`, `protobuf`), instead of `Boost` and `Threads`.

Reply:
Updated